### PR TITLE
Updates WAC FC tests to use new data

### PR DESCRIPTION
**Pull request by AustinSanders**
Updates WAC FC tests to use new data

## Related Issue
EIS

## Motivation and Context
Updates WAC FC tests to use data that has a view of Europa

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [x] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Review by scsides — DISMISSED** [review]

**Review by amystamile-usgs — COMMENTED** [review]
@acpaquette added a new test cube for WacFc at `/work/users/acpaquette/clipper`. Not sure if we should switch this test over to use that image instead or if it's the same as ClipperPho.cub

File: isis/tests/Fixtures.cpp [hunk 1]
```diff
@@ -1554,52 +1554,13 @@ namespace Isis {
...
-        BODY_CODE               = 502
-        BODY502_RADII           = (1562.6, 1560.3, 1559.5)
-        BODY_FRAME_CODE         = 10024
-        INS-159102_FOCAL_LENGTH = 150.40199
-        INS-159102_PIXEL_PITCH  = 0.014
-        INS-159102_TRANSX       = (0.0, 0.014004651, 0.0)
-        INS-159102_TRANSY       = (0.0, 0.0, 0.01399535)
-        INS-159102_ITRANSS      = (0.0, 71.404849, 0.0)
-        INS-159102_ITRANSL      = (0.0, 0.0, 71.4523)
-        INS-159102_OD_K         = (0.0, -0.0000538628307258204, -4.57142010849732E-09)
```
Comment (amystamile-usgs) [inline isis/tests/Fixtures.cpp:1597]: Just out of curiosity, does not updating the frame code here not cause the camera model to complain?

**Comment by amystamile-usgs** [comment]
It's also worth noting that the following tests are failing on dev. 
```	
1780 - Eis2Isis.Eis2IsisTestNacDefault (Failed)
1781 - Eis2Isis.Eis2IsisTestNacCheckline (Failed)
1783 - Eis2Isis.Eis2IsisTestWacDefault (Failed)

**Review by amystamile-usgs — APPROVED** [review]
👍

**Pull request merged**